x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_panel.tsx

```diff
@@ -86,11 +86,11 @@ export const IndexPatternDimensionTriggerComponent = function IndexPatternDimens
         }
         anchorClassName="eui-displayBlock"
       >
-        <EuiLink
+        <EuiText
```

Comment:
There are a few questions/concerns I have due to the change from `EuiLink` to `EuiText`:

- Since this element is no longer focusable, the styles applied by @mbondyra in a previous PR no longer set the focus styles (light blue background color behind text) appropriately for the child `.lnsLayerPanel__triggerLinkLabel` element. Can this be corrected?
- I believe this `aria-label` now no longer applies to this element. Can we move to the appropriate location?
- Should the `lnsLayerPanel__triggerLink` class name be changed, since it's no longer a link?